### PR TITLE
Add tags edition in card view

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: client/app/views/contacts/card.coffee [hunk 1]
```diff
@@ -64,32 +57,28 @@ module.exports = class ContactCardView extends Mn.LayoutView
...
-            @showChildView name, new DatapointsView
-                collection:    @model[name]
-                name:          name
-                cardViewModel: @model
+    _deleteBehavior: ->
+        opts = name: @options.model.get 'fn'
 
-        @addRegion 'xtras-infos', '[data-type=xtras-infos]'
-        @showChildView 'xtras-infos', new XtrasView model: @model
+        event:      'delete'
```
Comment (aenario) [inline client/app/views/contacts/card.coffee]: please explicit this is a return value `return deleteModal = `

File: client/app/views/contacts/components/data.coffee [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    initialize: ->
+        Mn.bindEntityEvents @model, @, @model.viewEvents
+
+
+    serializeData: ->
+        _.extend {}, super, fullname: @model.toString pre: '<b>', post: '</b>'
+
+
+    _renderDatapoints: ->
+        @$('.datapoints').each (index, el) =>
```
Comment (aenario) [inline client/app/views/contacts/components/data.coffee]: Not sure what you are doing there, but aren't you pulling data from the DOM ? Could you instead have datapoints as an array somewhere and use the array to contruct the DOM ?

**Comment by frankrousseau** [comment]
@m4dz could you rebase your PR?

File: client/app/views/contacts/card.coffee (hunk 1 above)
Comment (m4dz) [inline client/app/views/contacts/card.coffee]: I'm not a big of declaring useless `var` in scope, but can it be a better method name, like `_getDeleteModalCfg`?

File: client/app/views/contacts/components/data.coffee (hunk 2 above)
Comment (m4dz) [inline client/app/views/contacts/components/data.coffee]: Yeah, sure! The complex `DataView` should be written as `CompositeView` instead, to keep datapoints injection more consistent. Could we add it as a revamp issue for later, and keep it as it for this PR? (but I totally agree with you :smile:)

Comment (frankrousseau) [inline client/app/views/contacts/components/data.coffee]: Ok, let's do it later. But can you open an issue about it?

File: client/app/views/contacts/card.coffee (hunk 1 above)
Comment (frankrousseau) [inline client/app/views/contacts/card.coffee]: I think that Romain is mainly asking for the use of the `return` keyword.

Comment (m4dz) [inline client/app/views/contacts/card.coffee]: ooh… right, ok :smile:

**Comment by m4dz** [comment]
> could you rebase your PR?

Done.

**Comment by frankrousseau** [comment]
Thank you @m4dz

**Pull request merged**